.circleci/config.yml

```diff
@@ -87,6 +87,29 @@ jobs:
             source env/bin/activate
             python manage.py test
 
+  staging_deploy:
+    docker:
+      - image: ${ECR_ENDPOINT}/cloud-platform/tools:circleci
```

Comment:
📝 We have it in mind to update the usage of `ECR_DOCKER_REGISTRY` in the `build` job as it has the same contents as `ECR_ENDPOINT`.